FogCreek/Glitch-Community
# Whispering handball: My Stuff Design Updates

## Links
* Remix link: https://whispering-handball.glitch.me
* Issue-tracker link: 
- adding mediaQuery for prefers-reduced-motion https://app.zenhub.com/workspaces/community-5d07d1359dee6059a9688176/issues/fogcreek/glitch-community-work/318
- adding hover styling to bookmark button https://github.com/FogCreek/Glitch-Community-Work/issues/314
- halo animation not always completing https://github.com/FogCreek/Glitch-Community-Work/issues/310

## GIF/Screenshots:
- Reduced motion - eliminates hovering in animation on bookmark icon, halo animation
![reduced-motion](https://user-images.githubusercontent.com/519336/63049560-f83b5f80-bea6-11e9-91b6-77b76d110879.gif)

- Hover state - applies tertiary border to bookmark flag, fill to plus symbol
![hover](https://user-images.githubusercontent.com/519336/63049585-05f0e500-bea7-11e9-9db2-237029d7869d.gif)

- halo animation is not fully fixed, but I reduced the SVG size by 40% to try to optimize.  I think it just has to do with loading on pages that take a long time to load more generally, where everything is quite slow (such as https://glitch.com/search?q=hello)

## Changes:
* Added prefers-reduced-motion media query to `bookmark-button.styl`
* Edited Bookmark hover state
* Optimized halo SVG to reduce size in effort to improve animation
* Also fixed a bug that led to the `something went wrong` notification when clicking the bookmark button for featured projects

## How To Test:
* to turn on prefers-reduced-motion - follow directions here https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-motion
* try hovering over a project to see the new hover state of the bookmark button
* try bookmarking on user pages with lots of content 
* check that you don't get an error when you try to click the bookmark button for featured projects
